src/reducers/__tests__/query.spec.jsx

```diff
@@ -486,8 +515,6 @@ describe( 'reducer:query', () => {
             has_narrative: true
           }
           expect( target( state, action ) ).toEqual( {
-            enablePer1000: true,
-            mapWarningEnabled: true,
```

Comment:
has_narrative is a filter

Reply:
I just added a test for when it's in map mode.